.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
                         - rpm
 install:
         - ./autogen.sh
-        - ./configure --prefix=$HOME
+        - if [ "$TRAVIS_OS_NAME" = "linux" and "$CC" = "gcc"]; then ./configure --prefix=$HOME --enable-debug; else ./configure --prefix=$HOME; fi
```

Comment:
I think we want the enable-debug test to be an additional test, rather than replacing one of the current ones.